code/modules/ruins/spaceruin_code/forgottenship.dm

```diff
@@ -0,0 +1,115 @@
+	icon_state = "paper_words"
+	item_state = "paper"
+
+/obj/item/paper/fluff/ruins/forgottenship/powerissues
+	name = "Power issues"
+	info = "Welcome to battle cruiser SCSBC-12!<br>Our most advanced systems allow you to fly in space and never worry about power issues!<br>However, emergencies occur, and in case of power loss, <b>you must</b> enable emergency generator using uranium as fuel and enable turrets in bridge afterwards.<br><br><b>REMEMBER! CYBERSUN INDUSTRIES ARE NOT RESPONSIBLE FOR YOUR DEATH OR SHIP LOSS WHEN TURRETS ARE DISABLED!</b><br><br>Cyber Sun Industries (C) 2484."
+
+obj/item/paper/fluff/ruins/forgottenship/missionobj
+	name = "Mission objectives"
+	info = "Greetings, operatives. You are assigned to SCSBC-12(Syndicate Cyber Sun Battle Cruiser 12) to protect our high-ranking officer while he is on his way to next outpost. While you are travelling, he is the captain of this ship and <b>you must</b> obey his orders.<br><br>Remember, disobeying high-ranking officer orders is a reason for termination."
+
+///////////	forgottenship areas
+
+/area/ruin/space/has_grav/syndicate_forgotten_ship
+	name = "Syndicate Forgotten Ship"
+	icon_state = "syndie-ship"
+
+/area/ruin/space/has_grav/syndicate_forgotten_cargopod
+	name = "Syndicate Forgotten Cargo pod"
+	icon_state = "syndie-ship"
+
+/area/ruin/space/has_grav/powered/syndicate_forgotten_vault
+	name = "Syndicate Forgotten Vault"
+	icon_state = "syndie-ship"
+	noteleport = TRUE
+
+	//Cybersun hardsuit
+
+/obj/item/clothing/head/helmet/space/hardsuit/cybersun
+	name = "Cybersun hardsuit helmet"
+	desc = "Prototype hardsuit helmet with experimental armor plates, protecting from laser-based weapons very well, while giving limited protection against anything else."
+	icon_state = "cybersun"
+	item_state = "cybersun"
+	hardsuit_type = "cybersun"
+	armor = list("melee" = 25, "bullet" = 25, "laser" = 70, "energy" = 50, "bomb" = 15, "bio" = 100, "rad" = 50, "fire" = 60, "acid" = 60)
+	strip_delay = 600
+	actions_types = list()
+
+
+/obj/item/clothing/suit/space/hardsuit/cybersun
+	icon_state = "cybersun"
+	item_state = "cybersun"
+	hardsuit_type = "cybersun"
+	name = "Cybersun hardsuit"
+	desc = "Prototype hardsuit with experimental armor plates, protecting from laser-based weapons very well, while giving limited protection against anything else. Requires the user to activate the inner mechanism in order to unequip it, making it nearly impossible to take it off from somebody else."
```

Comment:
you should specify that it is still POSSIBLE to remove it from someone via the strip menu, even if it'll take a minute or so to do so (unlike, say, the abductor agent's suit if they haven't unlocked it)

Reply:
Sure thing, but that's just a description after all.